### DISCUSSION TITLE
How can I upgrade the version of docker-compose

### DISCUSSION THREAD

**Discussion by testworksau** [containers/podman#21526] [Q&A]
The docs explain how Podman Desktop can install the Compose engine:
https://podman-desktop.io/docs/compose/setting-up-compose

However, there is no guidance on how to upgrade the Compose engine.

The docker-compose binary has been installed at:

```
which docker-compose
> /Users/testworksau/.local/share/containers/podman-desktop/extensions-storage/podman-desktop.compose/bin/docker-compose
```

Is currently at version:

```
docker-compose version
> Docker Compose version v2.20.2
```

I also seem to have lost my Docker Compose widget at the bottom of my Podman Desktop application, which is where I thought I might be able to upgrade it.

I'm running Podman Desktop 1.7.0 on an M1 mac:

```
Client:       Podman Engine
Version:      4.9.2
API Version:  4.9.2
Go Version:   go1.21.6
Git Commit:   f9a48ebcfa9a39144be0f86f4ba842752835f945
Built:        Sat Feb  3 08:27:01 2024
OS/Arch:      darwin/arm64

Server:       Podman Engine
Version:      4.8.3
API Version:  4.8.3
Go Version:   go1.21.5
Built:        Thu Jan  4 01:10:40 2024
OS/Arch:      linux/arm64
```

**Comment by jeffmaury** [comment]
If you go to Settings -> CLI Tools you should be offered to update in the Compose card

**Reply by testworksau** [reply]
Thanks @jeffmaury - I only see `v2.20.2` with `No updates` despite `2.24.5` being available.

<img width="1042" alt="image" src="https://github.com/containers/podman/assets/11973031/74e13ff7-f656-4d57-b53a-a85ccefd5a93">

**Reply by jeffmaury** [reply]
I cannot reproduce on Windows but got the same on MacOS

**Reply by jeffmaury** [reply]
on MacOS seems that was another issue. I was able to install v2.24.1 and then an updated was proposed to me.

**Reply by testworksau** [reply]
Interesting; @jeffmaury did you just install via brew?

**Reply by jeffmaury** [reply]
No I used Podman Desktop to install docker-compose
